learning_journal/security.py

```diff
@@ -47,7 +47,7 @@ def includeme(config):
     config.set_authorization_policy(authz_policy)
     config.set_root_factory(NewRoot)
     # Session stuff for CSRF Protection
```

Comment:
This note is not really needed, unless you want to keep it there for your own benefit